Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/ConfigurationSideBarComponents/NavSideBarNode.js

```diff
@@ -77,15 +74,7 @@ export default function NavSideBarNode(props) {
                         </Button>
                     </Grid>
 
-                    <Grid item xs={2} className={classes.deleteButtonContainer}>
-                        <Button
-                            className={classes.deleteButton}
-                            color="primary"
-                            onClick={handleDelete}
-                        >
-                            <DeleteForeverIcon />
-                        </Button>
-                    </Grid>
+                    <DeletePopupSidebar id={id} deleteByID={deleteByID} />
```

Comment:
Nice job working with the current implementation!

Reply:
Fixed! There are only two files now, one for a generic delete warning and the other is a warning for deleting an entire scenario and all its data.